include/quakembd.h

```diff
@@ -72,6 +72,6 @@ void qembd_udelay(uint32_t us);
 void *qembd_allocmain(size_t size);
 int qembd_main(int c, char **v);
 int qembd_dequeue_key_event(key_event_t *e);
-int qembd_get_current_position(mouse_position_t *position);
+int qembd_get_mouse_motion(mouse_motion_t *motion);
```

Comment:
Rename to `qembd_get_mouse_event`.

Reply:
How about `qembd_get_mouse_movement`? because there are two different types of mouse events and because it's actually not just one event but an accumulation of all prior events, it might not be intuitive.